Resources/Prototypes/_RMC14/Voice/speech_emotes.yml

```diff
@@ -2,6 +2,12 @@
   id: Hiss
   name: rmc-emote-name-hiss
   category: Vocal
+  whitelist:
+    components:
+    - Vocal
+  blacklist:
+    components:
+    - BorgChassis
```

Comment:
There's probably a better way to do this, I'm not sure why you're blacklisting the borg chassis component?
You should probably look into how it's prevented on upstream emotes?

Reply:
I copied it from the other upstream emotes, they all whitelist vocal and blacklist borgchasis (though that's probably not needed here in this case).

The whitelist prevents the emotes from being used and showing up for ghosts, since ghosts don't have the vocal component.